corehq/apps/api/user_updates.py

```diff
@@ -176,42 +176,60 @@ def _update_location(user, location_object, user_change_logger):
     if primary_location_id is None and location_ids is None:
         return
 
-    user_current_primary_location_id = user.location_id
+    user_current_primary_location_id = user.get_location_id(user.domain)
+    user_current_locations = user.get_location_ids(user.domain)
```

Comment:
nit: I think you can remove `user_` from the front of these names as we are operating in the context of updating locations on a specific user so it is already clear.

Reply:
Removed in 30c498d7481b28020502f54341ff09c4866ffe20